src.mk

```diff
@@ -364,12 +364,11 @@ TEST_LIB_SOURCES =                                              \
   test_util/testutil.cc                                         \
   utilities/cassandra/test_utils.cc                             \
 
-FOLLY_SOURCES = \
-  third-party/folly/folly/detail/Futex.cpp                                     \
-  third-party/folly/folly/synchronization/AtomicNotification.cpp               \
-  third-party/folly/folly/synchronization/DistributedMutex.cpp                 \
-  third-party/folly/folly/synchronization/ParkingLot.cpp                       \
-  third-party/folly/folly/synchronization/WaitOptions.cpp                      \
+FOLLY_SOURCES =                                                 \
```

Comment:
IIUC, this is required only for the test-only simulation right? if we use a folly feature for Meta internal purposes, and it's guarded by something like ```#if defined(USE_FOLLY && FBCODE_ONLY)```, we'd just specify a dependency in the TARGETS file.

Reply:
That's right. The issue that is blocking this PR now is that our internal crash tests use our own Makefile builds (which is dubious for maximizing similarity to internal buck builds) and it's proving tricky to get this Makefile build working with internal folly. And that is pushing me toward moving us toward using buck builds for internal crash tests, and that's a bit of work.